packages/herdtools7/herdtools7.7.55/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+synopsis: "The herdtools suite for simulating and studying weak memory models"
+maintainer: "Luc Maranget <Luc.Maranget@inria.fr>"
+authors: [
+  "Luc Maranget <Luc.Maranget@inria.fr>"
+  "Jade Alglave <j.alglave@ucl.ac.uk>"
+  "Vincent Jacques <vincent@russian-dolls-sunflowers.com>"
+]
+homepage: "http://diy.inria.fr/"
+bug-reports: "http://github.com/herd/herdtools7/issues/"
+doc: "http://diy.inria.fr/doc/index.html"
+dev-repo: "git+https://github.com/herd/herdtools7.git"
+build: ["sh" "./dune-build.sh" "%{prefix}%"]
+install: ["sh" "./dune-install.sh" "%{prefix}%"]
+# @todo Add "build-doc" field
+# @todo Add "build-test" field
+depends: [
+  "ocaml" {>= "4.02.0"}
+  "dune"
```

Comment:
```suggestion
  "dune" {>= "1.2"}
  "ocamlfind" {build}
```
According to your `dune-project` file, this package requires at least dune 1.2. Furthermore, since you're using `dune install`, `ocamlfind` should be installed in the current switch to avoid problems (see https://github.com/ocaml/dune/issues/3025)